301-jenkins-acr-spinnaker-k8s/azuredeploy.json

```diff
@@ -1,350 +1,338 @@
-      "apiVersion": "2016-01-01",
-      "location": "[resourceGroup().location]",
-      "sku": {
-        "name": "Standard_LRS"
-      },
-      "kind": "Storage",
-      "properties": {
-        "encryption": {
-          "services": {
-            "blob": {
-              "enabled": true
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "User name for the Virtual Machine."
             }
-          },
-          "keySource": "Microsoft.Storage"
-        }
-      }
-    },
-    {
-      "name": "[variables('acrName')]",
-      "type": "Microsoft.ContainerRegistry/registries",
-      "location": "[resourceGroup().location]",
-      "apiVersion": "2016-06-27-preview",
-      "dependsOn": [
-        "[resourceId('Microsoft.Storage/storageAccounts', variables('acrStorageAccountName'))]"
-      ],
-      "properties": {
-        "adminUserEnabled": false,
-        "storageAccount": {
-          "name": "[variables('acrStorageAccountName')]",
-          "accessKey": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('acrStorageAccountName')), '2016-01-01').keys[0].value]"
+        },
+        "sshPublicKey": {
+            "type": "string",
+            "metadata": {
+                "description": "Configure all linux machines with the SSH public key string.  Your key should include three parts, for example 'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm'"
+            }
+        },
+        "devopsDnsPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the combined Jenkins and Spinnaker Virtual Machine."
+            }
+        },
+        "servicePrincipalAppId": {
+            "type": "string",
+            "metadata": {
+                "description": "Service Principal App ID (also called Client ID) that has contributor rights to the subscription used for this deployment. It is used by the Kubernetes cluster to dynamically manage resources (e.g. user-defined load balancers)."
+            }
+        },
+        "servicePrincipalAppKey": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Service Principal App Key that has contributor rights to the subscription used for this deployment. It is used by the Kubernetes cluster to dynamically manage resources (e.g. user-defined load balancers)."
+            }
+        },
+        "gitRepository": {
+            "type": "string",
+            "metadata": {
+                "description": "The URL to a public git repository used for the default Jenkins job. It must include a Dockerfile at the root of the repo."
+            },
+            "defaultValue": "https://github.com/lwander/spin-kub-demo.git"
```

Comment:
It's the same repo used by the official Spinnaker documentation, so yes:

http://www.spinnaker.io/docs/kubernetes-source-to-prod